main.py

```diff
@@ -138,7 +138,7 @@ async def on_message(message: discord.Message):
                 pass
 
             # don't respond to bot's own posts or if message reacts are disabled
-            if message.author == self.client.user or not message_reacts_enabled:
+            if message.author == self.client.user or not message_reacts_enabled or message.author.bot:

```

Comment:
`if (message.author == self.client.user) or (not message_reacts_enabled) or message.author.bot`

Reply:
no

Reply:
MY EYES